### PR TITLE
375 range percent values 26 50 not handled correctly in tada convertspecialchars

### PR DESCRIPTION
**Pull request by hillarymarler**

### CODE REVIEW

File: R/Utilities.R [hunk 1]
```diff
@@ -299,6 +299,10 @@ TADA_CheckColumns <- function(.data, expected_cols) {
 #' @param .data A TADA profile object
 #' @param col A character column to be converted to numeric
 #'
+#' @param percent.ave Boolean argument; default is clean = TRUE. When clean = TRUE,
```
Comment (cristinamullin) [inline R/Utilities.R:302]: The clean argument mentioned here in the documentation should be percent.ave

**Comment by hillarymarler** [comment]
@cristinamullin I fixed the documentation error.

**Comment by hillarymarler** [comment]
@cristinamullin - I will review this again and run some more examples tomorrow morning.

**Comment by hillarymarler** [comment]
@cristinamullin  - this is ready for you to review

**Comment by cristinamullin** [comment]
the DO full join is causing tada DR to print the join columns
image

We could use suppressMessages but if possible it is better to specify the "by" argument to avoid printing this message. dplyr only prints a message to let you know what its guess is for which columns to join by. In this case, we are joining by all columns. Adding a commit now to fix.

**Review by cristinamullin — APPROVED** [review]

**Pull request merged**